doc/dftb+/manual/dftbp.tex

```diff
@@ -2614,6 +2617,127 @@ \subsubsection{DftD3 optional settings}
   closely together.
 \end{description}
 
+\subsubsection{DftD4}
+\label{sec:dftbp.DftD4}
+
+The DFT-D4 dispersion correction in \dftbp{} is an implementation of the D4
+model by Stefan Grimme and coworkers. It is based on the method described in
+Ref.~\cite{caldeweyher-jcp-150-154122}.
+
+This dispersion correction for DFTB adds a contribution to the general
+Kohn--Sham-like energy
+\begin{equation*}
+  E_{\text{DFTB-D4}} = E_{\text{DFTB}} + E_{\text{disp}}
+\end{equation*}
+with $E_{\text{DFTB}}$ being the DFTB total energy and $E_{\text{disp}}$ the
+dispersion energy. The latter contains two-body and optional three-body
+contributions:
+\begin{equation*}
+  E_{\text{disp}} = E_{\text{disp}}^{(2)} + E_{\text{disp}}^{(3)}
+\end{equation*}
+
+The D4 model uses the Becke--Johnson damping function for two-body contributions:
+\begin{equation*}
+   E_{\text{disp}}^{(2)} = -\frac{1}{2} \sum_{A\neq B} \sum_{n=6,8,10} s_n
+   \frac{C_n^{AB}}{r_{AB}^n + f(R_0^{AB})}
+\end{equation*}
+with
+\begin{equation*}
+   f(R_0^{AB}) = a_1 R_0^{AB} + a_2 \text.
+\end{equation*}
+
+and the zero-damping function for three-body contributions:
+\begin{equation*}
+   E_{\text{disp}}^{(3)} = -\sum_{A}\sum_{\substack{B \\B < A}}\sum_{\substack{C\\ C < B}} s_9
+   \frac{  \left(3\cos\theta_A\cos\theta_B\cos\theta_C+1\right)\sqrt{C_6^{AB}C_6^{BC}C_6^{CA}}}{(r_{AB}r_{BC}r_{CA})^3} f^{(3)}(r_{AB})
+\end{equation*}
+with
+\begin{equation*}
+   f^{(3)} = \frac{1}{1 + 6\left(\frac{f(R_0^{AB})f(R_0^{BC})f(R_0^{CA})}{r_{AB}r_{BC}r_{CA}}\right)^{\alpha/3}}
+\end{equation*}
+
+In order to adjust the dispersion for various energy functionals, the choice of
+$s_8$ and the damping parameters $a_1$ and $a_2$ are
+treated as functional-dependent values. All other parameters are fixed based on
+these parameters.
+Depending on the choice of the $s_9$ parameter non-additive triple-dipole
+contributions will be evaluated. Including non-additive effects usually improves
+the description of dispersion interactions, but is also more expensive.
+
+As the DFTB energy functional is largely determined by the underlying
+parameterisation (the Slater--Koster-files) and the chosen DFTB model (e.g.\
+non-scc, scc, 3rd order, etc.), there are no universal parameter choices which
+can be used with all settings, but some relevant choices for various
+parameterisation are given in Appendix \ref{app:dftd4const}.
+
+\subsubsection{DftD4 optional settings}
+Apart from the , you can also adjust
```

Comment:
Seems to be a typo, what should be before the comma?

Reply:
Fixed